src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -81,15 +84,24 @@ export class MongoStorageAdapter {
     });
   }
 
-  // Used for testing only right now.
-  allCollections() {
-    return this.connect().then(() => {
-      return this.database.collections();
-    }).then(collections => {
+  // Delete all data known to this adatper. Used for testing.
+  deleteAllSchemas() {
```

Comment:
I think from the DB adapter point of view this is the correct name. SQL dbs, for example, won't have a schemas collection, and deleting the schema is actually the same thing as deleting all the data.